changelog/dmd.pragma-lib-local-path.dd

```diff
@@ -0,0 +1,37 @@
+`pragma(libLocal)` was added refer to a library in local path (relative to the module).
+
+This is useful to greatly simplify your build script, and makes it easier to distribute self-contained bindings to C libraries, or when you simply do not want to depend on `dub`.
+
+Example:
+
+
+demo/libs/library/package.d
+---
+module library;
+
+version (linux) {
+    pragma(libLocal, "bin/linux/library.a");
+}
+else version (Windows) {
+    pragma(libLocal, "bin/windows/library.lib");
```

Comment:
probably want \\ instead of /. Windows programs support / only erratically and unpredictably.